shell/platform/android/test/io/flutter/view/AccessibilityBridgeTest.java

```diff
@@ -1437,6 +1472,12 @@ protected void addToBuffer(
       updateString(increasedValue, increasedValueAttributes, bytes, strings, stringAttributeArgs);
       updateString(decreasedValue, decreasedValueAttributes, bytes, strings, stringAttributeArgs);
       updateString(hint, hintAttributes, bytes, strings, stringAttributeArgs);
+      if (tooltip == null) {
+        bytes.putInt(-1);
+      } else {
+        strings.add(tooltip);
+        bytes.putInt(strings.size() - 1);
```

Comment:
```suggestion
        bytes.putInt(tooltip);
```

Reply:
Can you elaborate? This int is for the index in the strings vector

Reply:
sorry. bad read